USING-CONFLUENCE.md

```diff
@@ -0,0 +1,69 @@
+# Using Confluence
+
+The confluence command uses the [Web API Confluence Dashboard](http://web-confluence.appspot.com/#!/) to generate new or improved browser compatibility data ready for submission in pull requests. The Web API Confluence Dashboard provides metrics and support data about web platform features across multiple browsers. The MDN Browser Compatibility Database provides a command for using Confluence to generate new and updated JSON files. This page explains how to do this.
```

Comment:
* "The confluence command" → "The `npm run confluence` command" (full command on first mention)
* The linked page doesn't actually use the word "Dashboard" so maybe "the Web API Confluence Dashboard" → "Web API Confluence"
* Sentence starting with "The MDN Browser Compatibility Database" can be removed entirely (reiterates the first sentence in this paragraph), then reorder to make the sentences go back together sensibly

Reworked as:

```suggestion
[Web API Confluence Dashboard](http://web-confluence.appspot.com/#!/) provides metrics and support data about web platform features across multiple browsers. The `npm run confluence` command uses Web API Confluence to generate new or improved browser compatibility data ready for submission in pull requests. This page explains how to do this.
```